pkg/cmd/server/start/start_master.go

```diff
@@ -378,7 +380,7 @@ func (m *Master) Start() error {
 	}
 
 	if m.api {
-		glog.Infof("Starting master on %s (%s)", m.config.ServingInfo.BindAddress, version.Get().String())
+		glog.Infof(MasterStartupMessage, m.config.ServingInfo.BindAddress, version.Get().String())
```

Comment:
hrmm, it makes it more painful to actually use the message and see the format string is being used correctly... not sure if `go vet` will catch format errors in referenced format strings like this